test/unit/test.speech_to_text.v1.js

```diff
@@ -344,4 +344,147 @@ describe('speech_to_text', function() {
       assert(isStream(speech_to_text.createRecognizeStream()));
     });
   });
+
+  describe('asynchronous callback api', function() {
+    it('should register new callback url', function() {
+      const path = '/v1/register_callback?callback_url=http%3A%2F%2Fwww.callback.fr%2Fresults&user_secret=ThisIsMySecret';
+      const response = {
+        status: 200,
+        url: 'http://www.callback.fr/results'
+      };
+
+      nock(service.url).persist().post(path).delay(100).reply(200, response);
+
+      const checkRes = function(err, res) {
+        assert.equal(JSON.stringify(err), JSON.stringify(null));
```

Comment:
this should just be `assert.ifError(err); - https://nodejs.org/api/assert.html#assert_assert_iferror_value